data/SkyPy/lsst-like.yml

```diff
@@ -34,7 +34,7 @@ tables:
       magnitudes: $blue.M
       coefficients: $blue.coeff
       filter:  bessell-B
-    mag_g, mag_r, mag_i, mag_z, mag_Y: !skypy.galaxies.spectrum.kcorrect.apparent_magnitudes
+    mag_g, mag_r, mag_i, mag_z, mag_y: !skypy.galaxies.spectrum.kcorrect.apparent_magnitudes
```

Comment:
is there a reason for the re-naming. Historically, the band Y is typed in capital letters. Not that I care much